homeassistant/helpers/config_validation.py

```diff
@@ -147,6 +147,20 @@ def entity_ids(value: Union[str, Sequence]) -> Sequence[str]:
     return [entity_id(ent_id) for ent_id in value]
 
 
+def entities_domain(domain: str):
+    """Validate that entities belong to domain."""
+    def validate(values: Union[str, Sequence]) -> Sequence[str]:
+        """Test if entitiy domain is domain."""
```

Comment:
Sorry for annoying you, but this is a spelling mistake (`entitiy` → `entity`)

Reply:
No, you are right. I'll fix that.